cmd/manager/main.go

```diff
@@ -31,6 +31,21 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/manager"
 	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
 	"sigs.k8s.io/controller-runtime/pkg/runtime/signals"
+
+	_ "k8s.io/code-generator/cmd/client-gen/args"
```

Comment:
@gildub @dymurray trying to understand why this change results in new imports in our manager main.go

Reply:
@djwhatle, those are development dependencies to build the project.

For instance the manifest rule of the Makefile uses `go run vendor/sigs.k8s.io/controller-tools/cmd/controller-gen/main.go`. 

By adding anonymous imports automatically includes those libraries which are used by CI too.

There might be different way of solving it.